src/components/dialog/demoBasicUsage/script.js

```diff
@@ -57,30 +56,19 @@ angular.module('dialogDemo1', ['ngMaterial'])
   };
 
   $scope.showAdvanced = function(ev) {
-    var useFullScreen = ($mdMedia('sm') || $mdMedia('xs'))  && $scope.customFullscreen;
-
     $mdDialog.show({
       controller: DialogController,
       templateUrl: 'dialog1.tmpl.html',
       parent: angular.element(document.body),
       targetEvent: ev,
-      clickOutsideToClose:true,
-      fullscreen: useFullScreen
+      clickOutsideToClose: true,
+      fullscreen: true  // only applies for screen size xs and sm
```

Comment:
This is a reasonable change. Please update then comment to (because this will actually show up in the demo)

```js
  fullscreen: true // Fullscreen only applies for -xs, -sm breakpoints.
```